### ISSUE TITLE
Write in process stdin

### ISSUE THREAD

**Issue by crgimenes** [nuveo/gofn#21]
I need to send some information to the container.
I do not want to map a volume, instead I want to send the information by typing in the stdin of the process.

Here is how I believe this feature should work:

```go
    gofn.Input = "some text"
    stdout, err := gofn.Run(...)
```

Internally gofn would pass the contents of the string to the stdin of the container.

At the end of the string it may be interesting to send an EOF character to signal that the string as ended and the process can start.

**Comment by crgimenes** [comment]
I'm starting to work on this feature today.
Suggestions are welcome!

**Comment by felipeweb** [comment]
closed via #35
